src/ui/StartUp.java

```diff
@@ -113,15 +105,13 @@ public static void main(String[] args) {
 	}
 
 	// starts the program
-	private void startProgram() {
-		//send the images array to ImageManager
-		this.imageManager.setImageList(images);
+	private void initializeImageProperties() {
```

Comment:
Metoda asta nu ar fi mai bine sa o mut in ImageManager si sa o _chem_ din StartUp gen: ImageManager.InitializeImageProperties() ?

Reply:
E o idee foarte buna!

Incearca acelasi lucru si cu metoda _buildImages_.